collect_app/src/main/res/values/typography.xml

```diff
@@ -32,7 +32,4 @@
         <item name="android:textAllCaps">false</item>
         <item name="android:letterSpacing">0.04</item>
     </style>
-
-    <style name="TextAppearance.Collect.Caption" parent="TextAppearance.MaterialComponents.Caption">
```

Comment:
We should only be introducing new styles there are overrides to the default, right? Like I said in https://github.com/getodk/collect/pull/4611#discussion_r652898601, there's quite a bit of chaos around where styles are introduced, where styles are used instead of theme attributes, etc. Might be worth a short narrative somewhere? I don't see anything currently but maybe it already exists in Android docs or something.

Reply:
Yeah you're right, there is no reason for us to have a text appearance style with no customization. For typography, it is actually pretty well documented at https://material.io/develop/android/theming/typography. There is also docs on color and shape theming there. I've done a bad job of enforcing this in reviews because my brain is reading the attribute and the explicit reference as the same thing. It only really becomes a huge deal when you move the component to another module where it won't have access to that style any longer.

I think we can maybe add to the `STATE.md` that we're using Material Theming but ultimately as we're using Material Components and will eventually switch to a Material Theme it's just the "correct" way of doing things within those systems, so it comes down to following the framework docs more than anything else.